### ISSUE TITLE
Elevated histidine in urine

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6040]
**Preferred term label:** Elevated histidine in urine

**Synonyms** Histidinuria

**Definition (free text, please give PubMed ID)** Concentration of histidine in the urine is above the normal range., PMID: 6796774

**Parent term (use hpo.jax.org/app)** Aminoaciduria HP:0003355

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is https://hpo.jax.org/app/browse/term/HP:0002927
I will add "Elevated histidine in urine" as a synonym.
